Public/Src/Utilities/Native/IO/Unix/FileUtilities.Unix.cs

```diff
@@ -668,7 +668,7 @@ public void SetFileAccessControl(string path, FileSystemRights fileSystemRights,
                 currentPermissions &= ~permissions;
             }
 
-            result = SetFilePermissionsForFilePath(path, currentPermissions);
+            result = SetFilePermissionsForFilePath(path, currentPermissions, followSymlink: false);

```

Comment:
>followSymlink: false) [](start = 77, length = 21)

maybe a comment for why we need to do this?

Reply:
will add